share/completions/cryptsetup.fish

```diff
@@ -1,4 +1,9 @@
 #cryptsetup 2.3.5
+function __fish_dm_names
+    set names (path basename /dev/mapper/*)
+    set -e names[(contains -i control $names)]
+    echo $names
```

Comment:
This won't work correctly because these need to be on separate *lines*.

You can just make this function

```fish
path basename /dev/mapper/* | string match -v control
```